src/CoverageTransformer.js

```diff
@@ -0,0 +1,246 @@
+const { Collector } = require('istanbul');
```

Comment:
If we're switching to ES6, we should probably switch to ESM rather than CJS? (I realize that may not be practical for now. Will need feedback from others (@kitsonk or @bryanforbes) if we're ok with this change.